features/built_in_matchers/class.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Class matchers
+
+  rspec-expectations a matcher to specify the nature of classes:
+
+    * `expect(Klass).to be_subclass_of(Klazz)`: calls `Klass.ancestors.include?(Klazz)`, which returns true if
+        Klazz is in Klass's class hierarchy or is a module and is included in a class in Klass's
+        class hierarchy. This matcher is aliased as `be_a_subclass_of` and `a_subclass_of`.
+
+
+  Scenario: be_(a_)subclass_of matcher
+    Given a file named "be_subclass_of_matcher_spec.rb" with:
+      """ruby
+      module Qux
+
+      end
+
+      class Foo
+
+      end
+      
+      class Bar < Foo
+
+      end
+
+      class Baz < Bar
+        include Qux
+      end
+
+      RSpec.describe Baz do
+        subject { Baz }
+
+        # the superclass
+        specify { expect(subject).to be_subclass_of(Bar) }
+
+        # the super-superclass
+        specify { expect(subject).to be_subclass_of(Foo) }
+
+        # Ruby internals
+        specify { expect(subject).to be_subclass_of(Object) }
+
+        # an included module
+        specify { expect(subject).to be_subclass_of(Qux) }
+
+        # negative passing case
+        it { is_expected.not_to be_subclass_of(String) }
+
+        # deliberate failures
+        specify { expect(subject).not_to be_subclass_of(Foo) }
+        specify { expect(subject).to be_subclass_of(Fixnum) }
```

Comment:
We consider explicitly referencing the `subject` from an example to be a a code smell:

http://blog.davidchelimsky.net/blog/2012/05/13/spec-smell-explicit-use-of-subject/

In addition, you've mixed `expect(subject)` with `is_expected` (which does the exact same thing). It would be good to stick with one style. Elsewhere we've tended to use `is_expected` for its terseness (to keep the focus on the matchers) but this case is slightly weird in that the default subject is an instance of `Baz` (thus forcing you to explicitly define it).  Given that, I think I might favor using `specify { expect(Baz).to ... }`.